test.js

```diff
@@ -9,6 +9,9 @@ test('Is loadable using requirejs', function (assert) {
 			'svgAsDataUri': 'function',
 			'svgAsPngUri': 'function',
 			'saveSvgAsPng': 'function',
+			'download': 'function',
+			'prepareSvg': 'function',
+			'saveSvg': 'function',
```

Comment:
Can you add any docs to the readme about running this test? I'd forgotten someone added it and I don't know how to use it.